examples/decentralized_protocol_config.json

```diff
@@ -0,0 +1,24 @@
+{
+	"environment": {
+		"type": "OrdinaryBandit",
+		"OrdinaryBandit": {
+			"arm": "BernoulliArm",
+			"means": [0.7, 0.5, 0.3, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
+		},
+		"protocol": "DecentralizedProtocol",
```

Comment:
Since `protocol` folder is put under the same directory with `environment`, I suggest put 'protocol' pars outside of `environment`.